packages/gluestick/src/renderer/helpers/matchRoute.js

```diff
@@ -15,8 +17,21 @@ module.exports = function matchRoute(
     const routes: Object = prepareRoutesWithTransitionHooks(
       getRoutes(store, httpClient),
     );
+
+    // React Router v3 does not catch the exceptions thrown by query-string,
+    // which causes 500 errors on pages to be shown
+    // We are handling it gracefully by omitting the query instead
+    let location: string;
+    try {
+      queryString.parse(req.url);
+      location = req.url;
+    } catch (e) {
+      const parsedUrl: Object = parseUrl(req.url);
+      location = parsedUrl && parsedUrl.pathname ? parsedUrl.pathname : '';
```

Comment:
`location = parsedUrl && parsedUrl.pathname || '';` should do the same, and avoids the redundancy of the ternary in this case.